Userland/Services/Taskbar/ClockWidget.cpp

```diff
@@ -20,9 +21,13 @@ ClockWidget::ClockWidget()
     set_frame_shadow(Gfx::FrameShadow::Sunken);
     set_frame_thickness(1);
 
-    m_time_width = font().width("22:22:22");
+    m_time_format_to_format_string.set("HH:MM:SS", "%T");
```

Comment:
Instead of the very loosely coupled assumption that one component will set one of these config settings to an arbitrary string, and then we interpret that arbitrary string in this other component. Maybe we can put this in some shared library between the two, and it can contain the possible config values and their properties (font width)?

That would add some rigidity and guard rails to this code.